classes/class06/twoter/package.json

```diff
@@ -27,11 +27,9 @@
     "mongoose": "^3.8.23",
     "mongoose-session": "0.0.3",
     "morgan": "^1.5.1",
-    "node-sass": "^2.0.1",
-    "node-sass-middleware": "^0.4.0",
     "passport": "^0.2.1",
+    "passport-facebook": "^1.0.3",
     "passport-local": "^1.0.0",
-    "sass-middleware": "^0.0.3",
     "serve-favicon": "^2.2.0",
     "sinon": "^1.12.2"
```

Comment:
A large portion of these dependencies only need to be in `dev-dependencies`. Your test frameworks don't need to be run in production.